packages/analytics/amplify_analytics_pinpoint_dart/lib/src/analytics_plugin_impl.dart

```diff
@@ -310,51 +247,5 @@ class AmplifyAnalyticsPinpointDart extends AnalyticsPluginInterface {
     _isConfigured = false;
     _autoEventSubmitter.stop();
     await _eventClient.close();
-    await _eventStorageAdapter.close();
-  }
-
-  /// Retrieve the stored pinpoint endpoint id
-  @visibleForTesting
-  Future<String> retrieveEndpointId({
-    required String pinpointAppId,
```

Comment:
Code moved to a new `EndpointIdManager` class.  All endpoint information retrieval will be refactored in this followup [PR](https://github.com/aws-amplify/amplify-flutter/pull/2639/files#diff-7415c6999da61d3f183ade45e81877137703db58cbab02ef3db08de3812eb9da) to consolidate data retrieval/management for Endpoints.